stoplight.js

```diff
@@ -0,0 +1,20 @@
+var isLightOn = false
+$(".bulb").click(function(){
+  if (isLightOn === false) {
+    $(this).addClass('active')
+    isLightOn = true
+  } else {
+    for ($(".bulb")).each(function(){
```

Comment:
Don't need to wrap the `.each` statement with `for`. Also when I load this there's a syntax error -- seems to be an extra `)`